Successfully updated:
Created Frame node eIaDpTY2N
Created Frame node R9iHQqCSk
Created Frame node u8iaKMCdi
Updated attributes for node eIaDpTY2N
Updated attributes for node R9iHQqCSk
Updated attributes for node u8iaKMCdi
Reordered node R9iHQqCSk within parent eIaDpTY2N to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -244,20 +244,51 @@
           <Description
               nodeId="yK6cCeTUB"
               width="1fr"
               height="fit-content"
               maxWidth="520px"
               <!-- project text style path, always starts with / -->
               inlineTextStyle="/Body lg"
           >
             Updated text 7692
           </Description>
+          <Stack
+              nodeId="eIaDpTY2N"
+              width="100%"
+              height="200px"
+              <!-- background color string or project color style path (if starts with /) -->
+              backgroundColor="rgb(50, 100, 200)"
+              borderRadius="12px"
+              layout="stack"
+              gap="20px"
+              padding="24px"
+              stackDirection="horizontal"
+              stackDistribution="center"
+              stackAlignment="center"
+          >
+            <Frame
+                nodeId="R9iHQqCSk"
+                width="100px"
+                height="100px"
+                <!-- background color string or project color style path (if starts with /) -->
+                backgroundColor="rgb(255, 255, 255)"
+                borderRadius="8px"
+             />
+            <Frame
+                nodeId="u8iaKMCdi"
+                width="150px"
+                height="80px"
+                <!-- background color string or project color style path (if starts with /) -->
+                backgroundColor="rgb(200, 200, 200)"
+                borderRadius="4px"
+             />
+          </Stack>
         </Header>
         <Actions
             nodeId="XQhemvRcY"
             width="fit-content"
             height="fit-content"
             layout="stack"
             gap="12px"
             stackDirection="horizontal"
             stackDistribution="center"
             stackAlignment="center"
